PendingReleaseNotes

```diff
@@ -65,3 +65,10 @@ Relevant tracker: https://tracker.ceph.com/issues/55521
   For more details, see:
 
   https://docs.ceph.com/en/latest/mgr/telemetry/
+* Cephadm may now be configured to carry out CephFS MDS upgrades without reducing
+``max_mds`` to 1. Previously, Cephadm would reduce ``max_mds`` to 1 to avoid
+having two active MDS daemons, which could lead to potential incompatibilities.
```

Comment:
"to avoid having active MDS daemons with different versions communicate with each other" ?

Reply:
When multiple MDS daemons are active, number of active MDS daemons cannot be reduced to 1 so easily. I think this line is a bit misleading. I'll change it to something better. Thanks for pointing this up.